h2o-docs/src/product/data-science/algo-params/offset_column.rst

```diff
@@ -1,7 +1,7 @@
 ``offset_column``
 -----------------
 
-- Available in: GBM, Deep Learning, GLM, GAM, CoxPH, XGBoost, Stacked Ensembles
+- Available in: GBM, Deep Learning, GLM, GAM, HGLM, CoxPH, XGBoost, Stacked Ensembles
```

Comment:
Please add the following algos to the end of Stacked Ensembles:

ANOVAGLM, ModelSelection